docs/virtual_machines/tekton_tasks.md

```diff
@@ -97,12 +97,11 @@ KubeVirt-specific Tekton tasks, which are focused on:
 All these tasks can be used for creating [pipelines](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md).
 SSP is creating multiple example pipelines, e.g.:
 
-- [Windows BIOS installer](https://github.com/kubevirt/ssp-operator/blob/main/data/tekton-pipelines/windows-bios-installer-pipeline.yaml) - Pipeline will prepare a template and Windows datavolume vith virtio drivers installed. User has to provide a link to working Windows 10 iso file. Pipeline is suitable 
-for Windows versions, which uses BIOS. More informations about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines/windows-bios-installer)
+- [Windows BIOS installer](https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/templates-pipelines/windows-bios-installer/manifests/windows-bios-installer.yaml) - Pipeline will prepare a template and Windows datavolume vith virtio drivers installed. User has to provide a link to working Windows 10 iso file. Pipeline is suitable for Windows versions, which uses BIOS. More information about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/templates-pipelines/windows-bios-installer/README.md)
```

Comment:
Correct link should be https://github.com/kubevirt/kubevirt-tekton-tasks/blob/main/release/pipelines/windows-bios-installer/windows-bios-installer.yaml.
The template-pipelines folder just contains templates. The result manifest are in release/pipelines.
Other links should point to release/pipelines too.